tools/run_tests/xds_k8s_test_driver/framework/test_app/client_app.py

```diff
@@ -344,18 +345,20 @@ def run(self,
         pod = self.k8s_namespace.list_deployment_pods(self.deployment)[0]
         self._wait_pod_started(pod.metadata.name)
         pod_ip = pod.status.pod_ip
+        rpc_port = self.stats_port
         rpc_host = None
 
         # Experimental, for local debugging.
         if self.debug_use_port_forwarding:
             logger.info('LOCAL DEV MODE: Enabling port forwarding to %s:%s',
                         pod_ip, self.stats_port)
-            self.port_forwarder = self.k8s_namespace.port_forward_pod(
+            self.local_forwarding_port, self.port_forwarder = self.k8s_namespace.port_forward_pod(
```

Comment:
Minor fix needed: unset `self.local_forwarding_port` in `cleanup()` after `self.k8s_namespace.port_forward_stop(self.port_forwarder)`

Reply:
`local_forwarding_port` removed, for the `PortForwarder` class.